lib/check_params.h

```diff
@@ -507,10 +507,6 @@ void printQudaInvertParam(QudaInvertParam *param) {
 
 #if defined(INIT_PARAM)
```

Comment:
Not here specifically, but in this file, `cuda_prec_eigensolver` needs to be appropriately init/checked.

Reply:
This is across `newQudaInvertParam`, `newQudaGaugeParam`, and `newQudaCloverParam`

(Sticking a pin in the thought that we should see if there's a way we can automate checking if each field in the structs have an appropriate entry in these functions.)

Reply:
Ah, also missed that it's needed for `reconstruct_eigensolver` as well.